### PR TITLE
Questions new

### PR DESCRIPTION
**Pull request by dmilburn**
This contains a controller and view for questions new and create.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  helper_method :current_user
+
+  def current_user
+    User.find_by(id: session[:user_id])
```
Comment (sgharms) [inline app/controllers/application_controller.rb:7]: :+1:

File: app/controllers/questions_controller.rb [hunk 2]
```diff
@@ -1,7 +1,27 @@
 class QuestionsController < ApplicationController
 
-
   def index
     @questions = Question.all
   end
+
```
Comment (sgharms) [inline app/controllers/questions_controller.rb:6]: Indentation is very broken.  Must fix.

File: app/controllers/questions_controller.rb [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+	def new
+		@question = Question.new
```
Comment (sgharms) [inline app/controllers/questions_controller.rb:8]: no

**Comment by dmilburn** [comment]
Crazy indentation is fixed. Wasn't appearing that way on my sublime which was weird. Thank you!

**Comment by sgharms** [comment]
Classic problem.  See: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=displaying%20tabs%20editor

**Pull request merged**